src/tests/ut/visit/visit_test.hpp

```diff
@@ -1,26 +1,40 @@
 #pragma once
 
+#include "doctor.hpp"
```

Comment:
detail: visit.hpp already includes "doctor.hpp" redundant, but pragma once should do its job anyway